portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches the hello message and adds it to the page.
+ */
+async function getHello() {
+    const response = await fetch('/data');
+    const hello = await response.text();
+    document.getElementById('hello-container').innerText = hello;
```

Comment:
`textContent` is preferred over `innerText`

There are a lot of details on why ([example](https://dev.to/4myc/javascript-innerhtml-innertext-and-textcontent-ih)), but tl;dr it's faster and potentially safer

Reply:
Thanks for letting me know, I didn't know textContent existed.